src/lib/_imports/trumps/u-summary/readme.md

```diff
@@ -0,0 +1,7 @@
+Utility classes for styling HTML `<summary>` elements.
+
+**Classes:**
+- `u-summary-with-a-link`: For summary element with a link inside
+- `u-summary-fake-marker`: For summary element with custom pseudo-element markers (▸/▾)
```

Comment:
Reference `src/lib/_imports/components/c-nav/readme.md` for how to show classes.